.vscode/settings.json

```diff
@@ -28,5 +28,13 @@ Happy linting! 💖
     "className",
     "ngClass",
     "classNameOverride"
+  ],
+  "tailwindCSS.experimental.classRegex": [
+    "classnames\\(([^)]*)\\)",
+    "(?:'|\"|`)([^']*)(?:'|\"|`)",
+    "classNames\\(([^)]*)\\)",
+    "(?:'|\"|`)([^']*)(?:'|\"|`)",
+    "csx\\(([^)]*)\\)",
```

Comment:
I think I've seen instances of `clsx` around (I'm assuming this is to search for what people name the import for `classnames`)

Reply:
noice, will quickly ammend